content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx

```diff
@@ -82,18 +82,16 @@ Here's how the following queries would match that text:
 
 ## Adding Indexes
 
-To speed up your full-text queries, you should add an index to your database.
-Prisma Migrate currently doesn't support adding search indices, but you can
-easily add one yourself.
+To speed up your full-text queries, you should add an index to your database. Prisma Schema currently doesn't support defining and Prisma Migrate doesn't support creating search indices, but you can easily add one yourself. The following example adds a `GIN` index to the table `posts` on the field `body`:
 
 ```sql
-CREATE INDEX post_body_index ON posts USING GIN (body);
+CREATE INDEX post_body_index 
+   ON posts USING GIN (to_tsvector('english', body));
```

Comment:
For a user setting up the index the first time, the Postgres extensions should be set up first.
```suggestion
CREATE EXTENSION pg_trgm;
CREATE EXTENSION btree_gin;
CREATE INDEX post_body_index 
   ON posts USING GIN (to_tsvector('english', body));
```

Reply:
Thanks @ruheni ! @janpio ,can we add Alex's section, and then merge?

Reply:
Unfortunately not, as the PR got conflicted in the meantime (seems some other changes were merged to the same file) so I would ask you @tanberry to recreate the PR on the current page please and then just close this one.

Reply:
I've resolved the conflict. You can take another look @janpio if everything is all good.